AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -65,8 +65,6 @@ - (void)setUp {
 - (void)tearDown {
 
   // Stop mocks.
-  [self.storageMock stopMocking];
-  [self.ingestionMock stopMocking];
```

Comment:
Should be reverted

Reply:
Same reason as above.

Reply:
As per [this comment](https://github.com/microsoft/appcenter-sdk-apple/pull/1996#discussion_r389114830), reverting back.